app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -1205,6 +1211,92 @@ class TopicLessonsFragmentTest {
     }
   }
 
+  @Test
+  fun testFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
```

Comment:
```suggestion
  fun testLessonPlayFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
```
Ditto for all other tests.

Reply:
Done.